public/assets/css/style.css

```diff
@@ -194,6 +194,37 @@ h4 {
     width: 500px;
 }
 
+/* BOUTON RETOUR TOP PAGE */
+#cRetour {
+    display: inline-block;
+    position: fixed;
+    bottom: 150px;
+    right: 20px;
+    width: 50px;
+    height: 50px;
+    background: url(../images/up-arrow.png), var(--third-color);
+    background-position: center;
+    background-size: 80%;
+    border-radius: 100%;
+    padding: 5px;
+    transition: all ease-in 0.5s;
+    backface-visibility: hidden;
+    -webkit-backface-visibility: hidden;
+}
+
+/* classe par défaut (invisible) du lien contenu dans la flèche de retour vers le haut */
+#cRetour.cInvisible {
+    bottom: -5vh;
+    opacity: 0;
+    transition: all ease-in 0.5s;
+}
+
+#cRetour.cVisible {
+    bottom: 150px;
+    right: 15px;
+    opacity: 1;
+}
```

Comment:
CSS Classenames and Ids should be in English

Reply:
translation done in an other commit, sorry !